Python/ceval.c

```diff
@@ -1045,8 +1041,8 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, _PyInterpreterFrame *frame, int
 #endif
 
     _PyCFrame cframe;
-    CallShape call_shape;
-    call_shape.kwnames = NULL; // Borrowed reference. Reset by CALL instructions.
+    _PyInterpreterFrame  pyframe;
```

Comment:
```suggestion
    _PyInterpreterFrame pyframe;
```

Also, consider renaming to `shim_frame` or `trampoline_frame`. A name like `pyframe` is way too overloaded to actually be helpful.